pkgs/applications/virtualization/podman/default.nix

```diff
@@ -44,6 +44,14 @@ buildGoModule rec {
     systemd
   ];
 
+  patches = [
+    # Temporary fix for darwin.  This works because the wrapper correctly sets the
+    # path.  When 4.0.0 is released, this goes away and the wrapper should set
+    # $CONTAINERS_HELPER_BINARY_DIR instead:
+    # https://github.com/containers/podman/blob/main/vendor/github.com/containers/common/pkg/config/config.go#L1171
+    ./0001-Look-for-gvproxy-in-PATH.patch
```

Comment:
I don't want to add this patch with `4.0.0` being released soon as it will complicate the update, we can deal with darwin after we've completed the upgrade.